src/libraries/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -49,8 +49,6 @@
     <Compile Include="System\Net\Sockets\OverlappedAsyncResult.cs" />
     <Compile Include="System\Net\Sockets\ReceiveMessageOverlappedAsyncResult.cs" />
     <Compile Include="System\Net\Sockets\UnixDomainSocketEndPoint.cs" />
-    <Compile Include="$(CommonPath)System\IO\StreamHelpers.CopyValidation.cs"
```

Comment:
Did this file get deleted? I see it being removed from a bunch of projects, but I don't see the actual file deletion. Or is still used somewhere?

Reply:
It's still used by a few OOB libraries that don't target netcoreapp.